plugins/services/src/js/columns/ServicesTableActionsColumn.tsx

```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { Trans } from "@lingui/macro";
+import { Cell, Icon } from "@dcos/ui-kit";
+import { Dropdown, Tooltip } from "reactjs-components";
+import { Hooks } from "#SRC/js/plugin-bridge/PluginSDK";
+import { SystemIcons } from "@dcos/ui-kit/dist/packages/icons/dist/system-icons-enum";
+import { iconSizeXs } from "@dcos/ui-kit/dist/packages/design-tokens/build/js/designTokens";
+
+import { isSDKService } from "../utils/ServiceUtil";
+import Pod from "../structs/Pod";
+import Service from "../structs/Service";
+import ServiceActionLabels from "../constants/ServiceActionLabels";
+import * as ServiceStatus from "../constants/ServiceStatus";
+import ServiceTree from "../structs/ServiceTree";
+import { ServiceActionItem } from "../constants/ServiceActionItem";
+
+const DELETE = ServiceActionItem.DELETE;
+const EDIT = ServiceActionItem.EDIT;
+const MORE = ServiceActionItem.MORE;
+const OPEN = ServiceActionItem.OPEN;
+const RESTART = ServiceActionItem.RESTART;
+const RESUME = ServiceActionItem.RESUME;
+const SCALE = ServiceActionItem.SCALE;
+const STOP = ServiceActionItem.STOP;
+
+function hasWebUI(service: Service | Pod | ServiceTree) {
+  return (
+    service instanceof Service &&
+    !isSDKService(service) &&
```

Comment:
I think this check needs to be removed - the latest conflict you had was because of the weburl icon change